pkg/template/servicebroker/test-scripts/bind.sh

```diff
@@ -16,6 +17,7 @@ curl \
   -X PUT \
   -H 'X-Broker-API-Version: 2.9' \
```

Comment:
Please check if sending X-Broker-API-Originating-Identity implies sending an X-Broker-API-Version > 2.9 and change across the board if required